### PR TITLE
update cennznet api and polkadot api version

### PR DESCRIPTION
**Pull request by KarishmaBothara**

### CODE REVIEW

File: packages/app-accounts/src/Overview.tsx [hunk 1]
```diff
@@ -22,17 +22,17 @@ type SortedAccount = { address: string; isFavorite: boolean };
...
+//   const ledger = getLedger();
+//
+//   try {
+//     const { address } = await ledger.getAddress();
+//
+//     keyring.addHardware(address, 'ledger', { name: 'ledger' });
+//   } catch (error) {
+//     console.error(error);
+//   }
+// }
```
Comment (ken-centrality) [inline packages/app-accounts/src/Overview.tsx:36]: Remove the comment block if we don't need it anymore

File: packages/app-accounts/src/Overview.tsx [hunk 2]
```diff
@@ -104,17 +104,17 @@ export default function Overview ({ className, onStatusChange }: Props): React.R
...
+        {/*    <Button.Or />*/}
+        {/*    <Button*/}
+        {/*      icon='question'*/}
+        {/*      isPrimary*/}
+        {/*      label={t('Query Ledger')}*/}
+        {/*      onClick={queryLedger}*/}
+        {/*    />*/}
+        {/*  </>*/}
+        {/*)}*/}
       </Button.Group>
```
Comment (ken-centrality) [inline packages/app-accounts/src/Overview.tsx:118]: Remove the comment block if we don't need it anymore

File: packages/app-explorer/src/BlockInfo/ByHash.tsx [hunk 3]
```diff
@@ -22,11 +23,60 @@ interface Props extends I18nProps {
...
-  const events = useCall<EventRecord[]>(api.query.system.events.at, [value]);
-  const getBlock = useCall<SignedBlock>(api.rpc.chain.getBlock, [value]);
-  const getHeader = useCall<HeaderExtended>(api.derive.chain.getHeader, [value]);
+  const mountedRef = useIsMountedRef();
+  const [[events, getBlock, getHeader], setState] = useState<[KeyedEvent[]?, SignedBlock?, HeaderExtended?]>([]);
+  // const [myError, setError] = useState<Error | null | undefined>(error);
+  // const events = useCall<KeyedEvent[]>(api.query.system.events.at, [value]);
+  // const getBlock = useCall<SignedBlock>(api.rpc.chain.getBlock, [value]);
+  // const getHeader = useCall<HeaderExtended>(api.derive.chain.getHeader, [value]);
+
```
Comment (ken-centrality) [inline packages/app-explorer/src/BlockInfo/ByHash.tsx:47]: Remove the comment block if we don't need it anymore

File: packages/app-explorer/src/BlockInfo/ByHash.tsx [hunk 4]
```diff
@@ -22,11 +23,60 @@ interface Props extends I18nProps {
...
+  //       [t('state'), 'start'],
+  //       [undefined, 'media--1200']
+  //     ]
+  //     : EMPTY_HEADER,
+  //   [getHeader, t]
+  // );
+
+  // const blockNumber = getHeader?.number.unwrap();
+  // const parentHash = getHeader?.parentHash.toHex();
+  // const hasParent = !getHeader?.parentHash.isEmpty;
```
Comment (ken-centrality) [inline packages/app-explorer/src/BlockInfo/ByHash.tsx:81]: Remove the comment block if we don't need it anymore

File: packages/app-settings/src/General.tsx [hunk 5]
```diff
@@ -101,17 +101,17 @@ export default function General ({ className, isModalContent, onClose }: Props):
...
+          {/*    <Dropdown*/}
+          {/*      defaultValue={ledgerConn}*/}
+          {/*      help={t('Manage your connection to Ledger S')}*/}
+          {/*      label={t('manage hardware connections')}*/}
+          {/*      onChange={_handleChange('ledgerConn')}*/}
+          {/*      options={ledgerConnOptions}*/}
+          {/*    />*/}
+          {/*  </div>*/}
+          {/*)}*/}
           <div className='ui--row'>
```
Comment (ken-centrality) [inline packages/app-settings/src/General.tsx:115]: Remove the comment block if we don't need it anymore

File: packages/apps/webpack.config.js [hunk 6]
```diff
@@ -120,74 +132,90 @@ function createWebpack ({ alias = {}, context, name = 'index' }) {
...
+      }),
+      // new HtmlWebpackPlugin({
+      //   inject: true,
+      //   template: path.join(
+      //     context,
+      //     `${hasPublic ? 'public/' : ''}${name}.html`
+      //   ),
+      //   PAGE_TITLE: 'CENNZnet Portal'
+      // }),
+      new webpack.optimize.SplitChunksPlugin(),
```
Comment (ken-centrality) [inline packages/apps/webpack.config.js]: Remove the comment block if we don't need it anymore

File: packages/react-api/src/index.ts [hunk 7]
```diff
@@ -18,8 +18,8 @@ export {
...
   withObservable,
-  clearLedger,
-  getLedger,
-  isLedger,
-  isLedgerCapable
+  // clearLedger,
+  // getLedger,
+  // isLedger,
+  // isLedgerCapable
 };
```
Comment (ken-centrality) [inline packages/react-api/src/index.ts:25]: Remove the comment block if we don't need it anymore

File: packages/react-api/src/ledger.ts [hunk 8]
```diff
@@ -1,59 +1,60 @@
...
+//
+//     assert(def, `Unable to find supported chain for ${api.genesisHash.toHex()}`);
+//
+//     if (def != null) {
+//       ledger = new Ledger(uiSettings.ledgerConn as 'u2f', def[1]);
+//     }
+//   }
+//
+//   return ledger as Ledger;
+// }
```
Comment (ken-centrality) [inline packages/react-api/src/ledger.ts:60]: Looks like the majority of `react-api/src/staking` files are commented out, worth to remove them to keep things tidy?

File: packages/react-components/src/Call.tsx [hunk 9]
```diff
@@ -6,24 +6,38 @@ import BN from 'bn.js';
...
+// export interface Props extends BareProps {
+//   children?: React.ReactNode;
+//   labelHash?: React.ReactNode;
+//   mortality?: string;
+//   onError?: () => void;
+//   value: IExtrinsic | IMethod;
+//   withHash?: boolean;
+//   tip?: BN;
+// }
+
```
Comment (ken-centrality) [inline packages/react-components/src/Call.tsx:30]: Remove the comment block if we don't need it anymore

File: packages/react-components/src/Call.tsx [hunk 10]
```diff
@@ -37,24 +51,70 @@ interface Value {
...
+    // }));
+    // const values = value.args.map((value): Value => ({
+    //   isValid: true,
+    //   value
+    // }));
+    // const hash = withHash
+    //   ? value.hash
+    //   : null;
+    //
+    // setExtracted({ hash, params, values });
```
Comment (ken-centrality) [inline packages/react-components/src/Call.tsx:117]: Remove the comment block if we don't need it anymore

File: packages/app-address-book/src/Address.tsx [hunk 11]
```diff
@@ -142,10 +142,15 @@ function Address ({ address, className, isFavorite, toggleFavorite }: Props): Re
...
-          name={isFavorite ? 'star' : 'star outline'}
+          color={isFavorite ? 'orange' : 'gray'}
+          icon='star'
           onClick={_onFavorite}
         />
+        {/*<Icon*/}
+        {/*  className={`${isFavorite && 'isSelected'}`}*/}
+        {/*  name={isFavorite ? 'star' : 'star outline'}*/}
+        {/*  onClick={_onFavorite}*/}
+        {/*/>*/}
```
Comment (ken-centrality) [inline packages/app-address-book/src/Address.tsx]: Remove the comment block if we don't need it anymore

File: packages/app-address-book/src/Address.tsx [hunk 12]
```diff
@@ -146,11 +146,11 @@ function Address ({ address, className, isFavorite, toggleFavorite }: Props): Re
...
-        {/*  className={`${isFavorite && 'isSelected'}`}*/}
-        {/*  name={isFavorite ? 'star' : 'star outline'}*/}
-        {/*  onClick={_onFavorite}*/}
-        {/*/>*/}
+        {/* <Icon */}
+        {/*  className={`${isFavorite && 'isSelected'}`} */}
+        {/*  name={isFavorite ? 'star' : 'star outline'} */}
+        {/*  onClick={_onFavorite} */}
+        {/* /> */}
       </td>
```
Comment (ken-centrality) [inline packages/app-address-book/src/Address.tsx:154]: Remove the comment block if we don't need it anymore

**Review by ken-centrality — CHANGES_REQUESTED** [review]
Looks good, there quite a few commented out code blocks, would be great if we can tidy them up before merge.

I noticed we've bumped up to use node `16`, then it's worth to update `.nvmrc` to `lts/gallium` to match with the requirement.

Also, does this resolve #170 and #172?

**Comment by KarishmaBothara** [comment]
> Looks good, there quite a few commented out code blocks, would be great if we can tidy them up before merge.
> 
> I noticed we've bumped up to use node `16`, then it's worth to update `.nvmrc` to `lts/gallium` to match with the requirement.
> 
> Also, does this resolve #170 and #172?

thanks for looking thro' the PR.. I removed commented code blocks and updated .nvmrc
Yes, updated the code to resolve #170 and #172

**Review by ken-centrality — APPROVED** [review]

**Comment by ken-centrality** [comment]
Looks good, but can you update .nvmrc to be `lts/gallium` instead of `16.0`, that will lock it to the LTS version of node 16, which is `16.13` atm

**Comment by KarishmaBothara** [comment]
> lts/gallium

thanks.. done

**Comment by KarishmaBothara** [comment]
> Looks good, but can you update .nvmrc to be `lts/gallium` instead of `16.0`, that will lock it to the LTS version of node 16, which is `16.13` atm

thanks done

**Comment by holygits** [comment]
@KarishmaBothara shall we merge?

**Comment by holygits** [comment]
need this so we can send txs to rata post update

**Comment by KarishmaBothara** [comment]
> @KarishmaBothara shall we merge?

yes.. wll check the last four steps of the pipeline.. not sure why status is waiting for that

**Pull request closed without merging**